superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -16,11 +16,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+/* eslint-disable camelcase */
```

Comment:
why do we need this?

Reply:
removed after fix filter immune related names.